pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a group of users (members), teams, and applications,
+with an access control system. Organization _members_ are balenaCloud users who may be grouped into _teams_, and can be assigned _roles_ such as developer, observer, or operator in balenaCloud applications. Organizations essentially improve visibility and reasoning over who has access to which applications and, consequently, to which device fleets.
+
+The feature is available for use in all balenaCloud accounts irrespective of plans. A balenaCloud account may contain one or more organizations.
```

Comment:
```suggestion
In balenaCloud, users organize around and collaborate in _organizations_. An organization is a group of users, teams and applications, and is managed by one or more organization administrators. An organization can have any number of members that can be grouped into teams and granted varying levels of access to its applications according to their assigned role.

Administrators are implicitly granted the ability to perform any possible action to the organization and its associated resources. This includes inviting and removing members, managing applications, teams and billing.

Each user can create or be a member of an unlimited number of organizations. An organization is automatically created upon user registration, and that user is assigned as its administrator. This organization is managed like any other, but currently cannot be deleted.

Billing applies to each organization separately. Administrators can at any time upgrade an organization to the appropriate billing plan as needed.
```

Sorry, I started trying to improve this blurb a bit but ended up with a bunch of text. Feel free to edit and split it up and distribute in the appropriate sections as you like.

I tried to refine the "organization" definition and highlight admins and billing.

Reply:
No worries, @dfunckt I will add all these suggestions in the batch and see them as a complete document. If I have any other changes will mention here.